### ISSUE TITLE
Example in Section 4.2.3.3 pc is invalid

### ISSUE THREAD

**Issue by yumaoka** [oasis-tcs/xliff-xliff-22#2]
The example in section 4.2.3.3 pc is not wellformed.

```
<unit id="1">
 <originalData>
  <data id="1">&lt;B&gt;</data>
  <data id="2">&lt;/B&gt;</data>
 </originalData>
 <segment><pc id="1" dataRefStart="1" dataRefEnd="2">Important</pc>
text</source>
 </segment>
</unit>
```

It should have `<source>` element after `<segment>`.

**Comment by rmraya** [comment]
Hello,

 

That example is indeed wrong. The opening tag for the  <source> element is missing.

 

Notice that the text published at GitHub is a port of the original specification for XLIFF 2.1. 

 

The example will be corrected on XLIFF 2.2. Thank you for reporting.

 

Regards,

Rodolfo

--

Rodolfo M. Raya        <mailto:rmraya@maxprograms.com> rmraya@maxprograms.com

Maxprograms      https://www.maxprograms.com

**Comment by rmraya** [comment]
Fixed in current draft
